include/opendht/http.h

```diff
@@ -57,6 +57,20 @@ using ConnectHandlerCb = std::function<void(const asio::error_code& ec,
 using ssl_socket_t = restinio::impl::tls_socket_t;
 using socket_t = asio::ip::tcp::socket;
 
+class OPENDHT_PUBLIC Url
```

Comment:
@aberaud  from what I understand, it should be called URL because it has a location ip:port and its algorithm is designed to extract these fields into the URL data structure with default target=/ , port=80 and protocol=http but if it is an URI I think it may perhaps, have different members or constructors added to it?
> A URI can be further classified as a locator, a name, or both. The term “Uniform Resource Locator” (URL) refers to the subset of URIs that, in addition to identifying a resource, provide a means of locating the resource by describing its primary access mechanism (e.g., its network “location”).  - https://danielmiessler.com/study/url-uri/